table/tables/tables_test.go

```diff
@@ -328,3 +328,42 @@ func (ts *testSuite) TestTableFromMeta(c *C) {
 	c.Assert(tb, IsNil)
 	c.Assert(err, NotNil)
 }
+
+func (ts *testSuite) TestPartitionAddRecord(c *C) {
+	createTable := `CREATE TABLE test.t1 (id int(11))
+PARTITION BY RANGE ( id ) (
+		PARTITION p0 VALUES LESS THAN (6),
+		PARTITION p1 VALUES LESS THAN (11),
+		PARTITION p2 VALUES LESS THAN (16),
+		PARTITION p3 VALUES LESS THAN (21)
+)`
```

Comment:
Maybe we need adding fourth partition `maxvalue`?

Reply:
Done.